moveos/moveos-stdlib/moveos-stdlib/sources/any_table.move

```diff
@@ -0,0 +1,148 @@
+/// Type of tables store any type of value.
+
+module moveos_std::any_table {
+
+    /// Type of tables
+    struct Table<phantom K: copy + drop> has store {
```

Comment:
any_table can satisfy all scenarios of table, consider directly replacing table with any_table under moveos-stblib? Or keep both?

Reply:
Keep both or implement table on any_table.